test-apps/display-test-app/src/frontend/App.ts

```diff
@@ -167,12 +167,6 @@ export class DisplayTestApp {
           uriPrefix: configuration.customOrchestratorUri || "http://localhost:3001",
           info: { title: "DisplayTestApp", version: "v1.0" },
         },
-        authConfig: {

```

Comment:
this is still needed for web case. Please remove this change

Reply:
My mistake.